### DISCUSSION TITLE
Value not in [empty] list

### DISCUSSION THREAD

**Discussion by jon-bondy** [Comfy-Org/ComfyUI#5531] [Q&A] [answered]
I am a total newbie.  I encountered this error message, and have no idea what is wrong.

![image](https://github.com/user-attachments/assets/e1e0206c-7d51-4e65-a7b0-baece129447b)

**Comment by LukeG89** [comment, marked as answer]
You simply don't have the models. Download them from [here](https://huggingface.co/Comfy-Org/stable-diffusion-3.5-fp8/tree/main/text_encoders) and place them inside `ComfyUI\models\text_encoders` folder.

Here's if you need more instructions: https://blog.comfy.org/sd3-5-comfyui/

**Comment by jon-bondy** [comment]
Thank you.  One set of instructions said to put them in the clip folder, not the text_encoders folder.

**Reply by LukeG89** [reply]
`clip` folder will be eventually removed in the future, they recently created the `text_encoders` one to replace it: https://github.com/comfyanonymous/ComfyUI/commit/ee8abf0cfff230286ac742138642c9876150f425

This is why I suggested it.
